sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKey.cs

```diff
@@ -162,12 +162,12 @@ public JsonWebKey(RSA rsaProvider, bool includePrivateParameters = default, IEnu
         /// <summary>
         /// Gets the RSA modulus.
         /// </summary>
-        public byte[] N { get; internal set; }
+        public byte[] N { get; set; }
```

Comment:
What happens in our serializer if these are null? Do we handle that case correctly? Make sure that we do. I would write a test case for at least public key components that cryptographic operations don't fail if null. In that case, we should either have a setter that validates a null value isn't assigned (probably not ideal since they default to null, so not all will be non-null on deserialization) or make sure our code handles those cases correctly (it might already - make sure we have coverage).

Reply:
It seems we have coverage for [E](https://github.com/Azure/azure-sdk-for-net/blob/2e0bc399ba202a7301b0144e19164102502c52f2/sdk/keyvault/Azure.Security.KeyVault.Keys/tests/JsonWebKeyTests.cs#L400) and [N](https://github.com/Azure/azure-sdk-for-net/blob/2e0bc399ba202a7301b0144e19164102502c52f2/sdk/keyvault/Azure.Security.KeyVault.Keys/tests/JsonWebKeyTests.cs#L412) being null.